dora/core/client/fs/src/main/java/alluxio/client/block/BlockStoreClient.java

```diff
@@ -103,265 +71,6 @@ public BlockInfo getInfo(long blockId) throws IOException {
     }
   }
 
-  /**
```

Comment:
I only removed the Block stream related methods from this client, while keeping a few methods here which are still used by status checks like `fsadmin report capacity`. Admittedly those status checks should be refactored, but not in this PR.